ClinGen/clincoded
# Connects to #1363. Connects to #1481. Connects to #1554. Connects to #1580.

**Steps to test #1363:**
1. In a Gene-Disease record, add a new **Individual** evidence.
2. In the **Individual** evidence curation form, select the **Yes** option from the **Is this Individual a proband** dropdown.
3. Then click the **Copy disease term from Gene-Disease Record** button to associate this Proband with a disease.
4. Scroll down the page, click the **Add ClinVar ID** button to associate this Proband with a variant.
5. Then in the **Individual -- Score Proband** panel, select the **Score** option from the **Select Status** dropdown and either one of the options from **Confirm Case Information type** dropdown.
6. Expect to see the **Explanation** text field being enabled now, in contrast to the previous behavior that it was ONLY enabled given a different score than the default score. (Note: This field is still a **required** field if a different score has been selected).
7. Now Change the **Select Status** dropdown option to **Review** and change the **Confirm Case Information type** dropdown option to **No Selection**.
8. Expect to see the **Explanation** text field being enabled without a **Confirm Case Information type** value. This is because a **Review** status does not require curators to select a case information type.
9. Now Change the **Select Status** dropdown option to **Contradicts** and expect to see the **Explanation** text field being shown and enabled, in contrast to the previous behavior that this status didn't provide an explanation box.
10. This updated **Explanation** behavior will also be consistent with the **Experimental** evidence, except that there is no **Confirm Case Information type** dropdown options available in **Experimental**.
11. This updated **Explanation** behavior will also be consistent with the **Individual** evidence in Gene-Disease records that have **Mode of Inheritance** that are not AD, AR or X-Linked. In those use cases, expect to see the **Explanation** text field being shown and enabled regardless of the **Score Status** and whether a **Case Information Type** option is selected.
***
**Steps to test #1481:**
1. In a Gene-Disease Record, add a **Proband** and **Experimental** evidence. Complete and score them.
2. Upon returning to the record page, click on the **Classification Matrix** button at the upper-right.
3. On the subsequent matrix page, expect to see an icon of blue circle with an "i" next to the **Replication Over Time (Yes/No)** table header in the classification table.
4. Mouseover to the icon and expect to see a tooltip bubble with descriptive text.
![image](https://user-images.githubusercontent.com/6956310/35647800-933c385c-0688-11e8-9c5c-3bd795408b35.png)
***
**Steps to test #1554:**
1. In a Gene-Disease Record, add a **Proband** evidence and give it a **0.1** score in contrast to the default score.
2. Upon returning to the record page, click on the **Classification Matrix** button at the upper-right.
3. On the subsequent matrix page, expect to see the **LIMITED** range being highlighted in blue and the **LIMITED** range now being **0.1-6** in the classification table.
![image](https://user-images.githubusercontent.com/6956310/35648156-9e6418fc-0689-11e8-88ee-87ee1d0dd282.png)
4. Click the **Save** button and proceed to the next **view-only** classification page. Expect to see the same behavior described above.
***
**Steps to test #1580:**
1. Start a VCI interpretation with the ClinVar ID **50000**.
2. Proceed to the **Case/Segregation** tab, and scroll the page to the **Allele data (cis/trans)** panel.
3. From the **PM3** dropdown, expect to see the new **PM3_Very Strong** option.
![image](https://user-images.githubusercontent.com/6956310/35648404-85002aee-068a-11e8-8328-36f0285c23a4.png)
